src/App.jsx

```diff
@@ -1,7 +1,37 @@
+import { Provider } from "react-redux";
+import Articles from "./components/Articles";
+import Footer from "./layouts/Footer";
+import Navbar from "./layouts/Navbar";
+import { store } from "./services/actions/store";
+import { BrowserRouter as Router, Route, Switch } from "react-router-dom";
+import PageNotFound from "./components/PageNotFound";
+
+import PubDetails from "./pages/PubDetails";
+import ModNavBar from "./layouts/ModNavBar";
+
 export default function App() {
   return (
-    <h1 className="text-3xl text-red-500 font-bold underline text-center ">
-     React Intermediate Project with Redux
-    </h1>
-  )
-}
\ No newline at end of file
+    <>
+      <Provider store={store}>
+        <Router>
+          <>
+            <Navbar />
+            <ModNavBar />
+          </>
```

Comment:
You can move these component outside the router component, this will allow you to avoid unnecessary empty fragment